spec/integrations/fixtures/rspec/let_it_be_fixture.rb

```diff
@@ -190,3 +190,17 @@
     end
   end
 end
+
+describe "does not leak", order: :defined do
```

Comment:
That scenario that helped to discover the issue. Slightly simplified compared to the original.